defaults/main.yml

```diff
@@ -53,6 +53,63 @@ neofs_ir__pprof_allow: []
 neofs_ir__pprof_group_allow: []
 neofs_ir__pprof_host_allow: []
 
+neofs_ir__blockchain_database_path: "{{ neofs_ir__data_dir }}/morph.bolt"
+neofs_ir__blockchain_notary_disabled: False
+neofs_ir__blockchain_time_per_block: '15s'
+neofs_ir__blockchain_magic: '735783775'
+neofs_ir__blockchain_seed_nodes: []
+neofs_ir__blockchain_rpc_addresses: []
+neofs_ir__blockchain_p2p_addresses: []
+
+neofs_ir__validators:
+      - '026fa34ec057d74c2fdf1a18e336d0bd597ea401a0b2ad57340d5c220d09f44086'
+      - '039a9db2a30942b1843db673aeb0d4fd6433f74cec1d879de6343cb9fcf7628fa4'
+      - '0366d255e7ce23ea6f7f1e4bedf5cbafe598705b47e6ec213ef13b2f0819e8ab33'
+      - '023f9cb7bbe154d529d5c719fdc39feaa831a43ae03d2a4280575b60f52fa7bc52'
+      - '039ba959e0ab6dc616df8b803692f1c30ba9071b76b05535eb994bf5bbc402ad5f'
+      - '035a2a18cddafa25ad353dea5e6730a1b9fcb4b918c4a0303c4387bb9c3b816adf'
+      - '031f4d9c66f2ec348832c48fd3a16dfaeb59e85f557ae1e07f6696d0375c64f97b'
+neofs_ir__consensus_committee: "{{ neofs_ir__validators }}"
+
+neofs_ir__external_sidechain: False
+neofs_ir__external_sidechain_config:
+    client:
+      - address: 'ws://localhost:40332/ws'
+neofs_ir__internal_sidechain_config:
+    magic: "{{ neofs_ir__blockchain_magic }}"
+    committee: "{{ neofs_ir__consensus_committee }}"
+    storage:
+      type: boltdb
+      path: "{{ neofs_ir__blockchain_database_path }}"
+    notary_disabled: "{{ neofs_ir__blockchain_notary_disabled }}"
+    time_per_block: "{{ neofs_ir__blockchain_time_per_block }}"
+    max_traceable_blocks: 2102400
+    seed_nodes: "{{ neofs_ir__blockchain_seed_nodes }}"
+    rpc:
+      listen: "{{ neofs_ir__blockchain_rpc_addresses }}"
+    p2p:
+      dial_timeout: 5s
+      proto_tick_interval: 2s
+      listen: "{{ neofs_ir__blockchain_p2p_addresses }}"
+      peers:
+        min: 1
```

Comment:
Where does it come from?

Reply:
From the sample config, both. What values should be there instead?

Reply:
Node's `config.yaml` is just a sample and what you have here in `main.yml` is something that looks like a mainnet/testnet config. So I'd expect some values suitable for testnet/mainnet, and they're the same as in NeoGo configs for these networks: https://github.com/nspcc-dev/neo-go/blob/c50ab951645a1c901315ac5366170868245c34fd/config/protocol.mainnet.yml#L62

Reply:
Updated